pom.xml

```diff
@@ -2,14 +2,14 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.amazonaws</groupId>
     <artifactId>dynamodb-lock-client</artifactId>
-    <version>1.2.0</version>
+    <version>1.2.1</version>
```

Comment:
Aren't changes to this file already merged under the PR: https://github.com/awslabs/amazon-dynamodb-lock-client/pull/60 ? I do not think they should show up in this PR

Reply:
Fixed, unsure why they showed up, just reverted the changes manually though